### DISCUSSION TITLE
ApplicationInsights-agent-3.4.12 files generated in Windows/Temp

### DISCUSSION THREAD

**Discussion by fred-sudo** [open-telemetry/community#1585] [Community Discussions]
for the spring boot-based batch app, added the above-mentioned agent for Azure monitor. it works fine. It does export data to azure monitor.  The side effect is every time the batch process starts it creates the otel-agentxxxxx/agent.jar  folder in Windows/Temp folder. which creates memory overflow issues. no json properties or method found (ex. ApplicationInsight.Attach()) to detach and delete generate folders and files.

**Comment by trask** [comment]
Hi @fred-sudo! I'd suggest opening an issue in https://github.com/microsoft/ApplicationInsights-Java

**Reply by fred-sudo** [reply]
Thank you..https://github.com/microsoft/ApplicationInsights-Java/issues/3183
